pages/404.tsx

```diff
@@ -0,0 +1,52 @@
+import { useRouter } from "next/router";
+import LandingFooter from "@/Components/LandingFooter";
+
+const NotFoundPage = () => {
+  const router = useRouter();
+  const navLinkStyle =
+    "font-archivo relative text-xl w-fit block after:block after:content-[''] after:absolute after:h-[3px] after:bg-black after:w-full after:scale-x-0 after:hover:scale-x-100 after:transition after:rounded-full after:duration-500 after:origin-center";
+  const handleGoBack = () => {
+    router.back();
+  };
+  const redirectToHome = () => {
+    router.push("/");
+  };
+  return (
+    <>
+      <div className="sticky top-0 z-20 w-full border-b-2 border-white bg-white/30 backdrop-blur-md dark:bg-neutral-950/50 dark:border-zinc-700">
```

Comment:
Why are we creating a Navbar? Don't we have a `Navbar` component that we can use here?

Reply:
This navbar is customised to have no navlinks and a button of Go Back. Since there was no component for this I copy pasted some code from the original nav and modified it.

Reply:
We can just put a simple condition in that component instead of copy-pasting the code. Just check if the page is `404` or whatever route you have given to the `404` page. if that's true hide whatever you want to hide.

Reply:
I have added the original Navbar code as requested, with a boolean prop named 'error404Check', according to which the displayed Navbar is changed for 404 page or else simple pages. @lalitmee

Reply:
Are any further changes requried?